### PR TITLE
#167765380 Integrate HoundCI to Github

### PR DESCRIPTION
**Pull request by okobaba1**
#### What does this PR do?
This PR integrates the houndCi with the github repository
#### Description of Task to be completed?
https://docs.google.com/document/d/1177UB0b7FJi4nV0mmronw4LorTCk8e_7a2-c4O45ca0/edit
#### How should this be manually tested?
It can be tested by making a pull requests
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#167765380](https://www.pivotaltracker.com/story/show/167765380)

### CODE REVIEW

**Review by uggoprince — APPROVED** [review]
@okobaba1  you integrated it well.
The next thing (if we are yet to) is we all to login to hound.com with our github account and activate phoenix-bn-backend repo, so as to be getting code check reports (If TTL and PO deems fit)

**Review by debelistic — COMMENTED** [review]
Great job,

The hound keeps telling me, some features are only allowed in ES6. Are those kinds of errors necessary?
Also, why not use `.eslintrc` instead of  `.eslintrc.js`. The extension is redundant

**Comment by okobaba1** [comment]
> Great job,
> 
> The hound keeps telling me, some features are only allowed in ES6. Are those kinds of errors necessary?
> Also, why not use `.eslintrc` instead of `.eslintrc.js`. The extension is redundant

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
+  ignore_file: .eslintignore
```
Comment (anneKay) [inline .hound.yml]: Add newline here

**Pull request merged**